dockstore-webservice/src/main/java/io/dockstore/webservice/resources/LambdaEventResource.java

```diff
@@ -61,15 +60,18 @@ public List<LambdaEvent> getLambdaEventsByOrganization(@ApiParam(hidden = true)
             @ApiParam(value = PAGINATION_OFFSET_TEXT) @QueryParam("offset") @DefaultValue("0") String offset,
             @ApiParam(value = PAGINATION_LIMIT_TEXT, allowableValues = "range[1,100]", defaultValue = PAGINATION_LIMIT) @DefaultValue(PAGINATION_LIMIT) @QueryParam("limit") Integer limit) {
         User authUser = userDAO.findById(user.getId());
-        List<Token> githubToken = tokenDAO.findGithubByUserId(authUser.getId());
-        if (githubToken.size() == 0) {
+        List<Token> githubTokens = tokenDAO.findGithubByUserId(authUser.getId());
```

Comment:
A little unrelated, but why is this a list anyway? 
It seems like users in our system should only have one github token?

Reply:
That, I do not know.